script/test

```diff
@@ -68,7 +68,7 @@ install() {
 get_openthread() {
   if ! [[ -x $OT_DIR ]]; then
     mkdir -p "$(dirname $OT_DIR)"
-    git clone https://github.com/openthread/openthread.git $OT_DIR
+    git clone https://github.com/simonlingoogle/openthread.git --depth 1 --branch otns/use-virtual-time-uart $OT_DIR
```

Comment:
Can not use OT master yet because it is forbidden to compile with both `OTNS=1` and `VIRTUAL_TIME_UART=1`